app/pages/multisig/AccountTransactions/UpdateAccountCredentials/UpdateCredentialsPage.tsx

```diff
@@ -301,14 +308,18 @@ function UpdateCredentialPage({ exchangeRate }: Props): JSX.Element {
             currentAccountInfo.accountCredentials
         ).map((accountCredential) => {
             const credentialIndex = parseInt(accountCredential[0], 10);
-            const cred = accountCredential[1].value.contents;
-            if (cred.regId) {
+            const cred = accountCredential[1].value;
+            // TODO: don't use literal string
```

Comment:
Why not?

Reply:
I was thinking it would be nicer if there was an enum, probably.

Reply:
But that would be the responsibility of the SDK to provide that. We can consider adding that later, but I don't think the TODO should be in the desktop wallet.

Reply:
I've removed it.